subprojects/core/src/main/java/org/gradle/api/internal/StartParameterInternal.java

```diff
@@ -46,8 +47,9 @@ public StartParameter newBuild() {
     protected StartParameter prepareNewBuild(StartParameter startParameter) {
         StartParameterInternal p = (StartParameterInternal) super.prepareNewBuild(startParameter);
         p.watchFileSystem = watchFileSystem;
-        p.watchFileSystemVerboseLogging = watchFileSystemVerboseLogging;
+        p.watchFileSystemDebugLogging = watchFileSystemDebugLogging;
```

Comment:
I wonder if `verbose` and `debug` should be at the same place in the two fields. So something like:
- `debugWatchFileSystemLogging` and `verboseVfsLogging`, or
- `watchFileSystemDebugLogging` and `vfsVerboseLogging`.

WDYT?

Reply:
Renamed.